packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -1136,6 +1138,17 @@ const TagBox = (SelectBox as any).inherit({
     }
   },
 
+  _setTagAria(element, value) {
+    const aria = {
+      role: 'button',
+      label: value,
+      // eslint-disable-next-line spellcheck/spell-checker
+      roledescription: messageLocalization.format('dxTagBox-tagRoleDescription'),
```

Comment:
cannot we just wrap `roledescription` with single quotes to get rid of the spellcheck error?

Reply:
No, because eslint rules.